gsd-web/README.md

```diff
@@ -76,6 +76,36 @@ To be able to use the sign in functionality locally, you'll need to create a Git
 
 You'll need to save three ENV variables in your terminal profile so that the application can properly load.
 
+| Key | Value | Usage |
+| --- | --- | --- |
+| GSD_SESSION_KEY | Secure random string | Seed for Session Security |
+| GSD_GITHUB_KEY | Github OAuth app client id | Provides app access to GH Api |
+| GSD_GITHUB_SECRET | OAuth app client secret |  |
+
+Only ENV variables prefixed with `GSD_` will passed to the Quasar environment.
+
+__ENV Load Order__
+1. Hardcoded defaults
+2. (Optional) .env values
+3. Shell ENV prefixed with `GSD_`
+
+##### Project local .env file
+
+This project uses `dotenv` to simplify management of required settings and ENV variables.
+
+You will find an `.env.sample` in the project root and by renaming it to `.env` will cause it to load before quasar boots.
+
+For convenience you can run the following for an OS agnostic setup
+```bash
+yarn devinit
+```
```

Comment:
I think we should simplify this step by adding a guard clause to the init command, and simply running it on every `yarn dev` call. The additional boot time added should be negligible, and it would simplify the process for new folks coming in.

Reply:
One thing that I could see tripping up newer developers is renaming the `.env.sample` to `.env` and then committing the "deleted `.env.sample`" change by accident.